apps/sms/views/conversation/js/conversation.js

```diff
@@ -1180,7 +1187,7 @@ var ConversationView = {
         // Thread-less drafts are orphaned at this point
         // so they need to be resaved for persistence
         // Otherwise, clear the draft directly before leaving
-        if (!Threads.currentId) {
+        if (!this.currentThreadId) {
```

Comment:
so I mean: `if (!Threads.currentId)` really means `if (Navigation.isCurrentPanel('thread'))`; but because we did the support for "drafts" before the `Navigation` object existed, we had to check the panel using `Threads.currentId` instead of using a clean method on `Navigation`.

So it was only a comment saying this :) hope this is clearer